acquia_lift.batch.inc

```diff
@@ -119,20 +116,27 @@ function acquia_lift_batch_finished($success, $results, $operations) {
   }
 }
 
-/**
- * Gets the required operations for syncing the specified agents to Lift.
- *
- * @param $agents
- *   An array of agents with agent names as keys and stdClass objects rep-
- *   resenting the agents as values.
- * @return array
- *   An array of operation arrays, each with a 'method' key and an 'args'
- *   key, that can be processed by acquia_lift_sync_item().
- */
-function acquia_lift_get_sync_operations_for_agents($agents) {
+function acquia_lift_get_sync_operations_for_personalization($personalization) {
```

Comment:
Docblock sync-up... but I think this is the first time we're introducing "personalization" as a code term rather than just in text.  I think we should probably set some expectation around the words agent, campaign, and personalization within the code.  I worry it will soon be at that point when someone asks what a function is and we have to give them the history of the product in order to explain it.